src/jobs/index.html

```diff
@@ -61,201 +61,55 @@
-        <li>Know how to have fun while staying professional and communicating clearly.</li>
-        <li>Follow the latest trends, styles, and aesthetics.</li>
-        <li>Have experience in both digital and print environments—websites, social media, merchandise (clothing, coffee mugs, gift cards), and promotional materials.</li>
-        <li>Be passionate about crypto - you don’t have to know all the tech, but your interest in the space should be obvious.</li>
-        <li>Work efficiently while being remote.</li>
-        <li>Have up-to-date software (Adobe Photoshop/Illustrator & Sketch).</li>
-      </ul>
-      <h5>You Will:</h5>
-      <ul>
-        <li>Own the design direction for MyCrypto’s merch / social media / etc.</li>
-        <li>Influence the design direction for MyCrypto’s brand.</li>
-        <li>Create imagery for blogs & social media.</li>
-        <li>Provide design guidance on our product.</li>
-        <li>Coordinate with engineers and outside vendors on an as-needed basis.</li>
-        <li>Interact, communicate, and present ideas with various teams.</li>
-        <li>Work efficiently and be mindful of costs and deadlines.</li>
-      </ul>
-      <h5>If interested, please email the following to <a target="_blank" rel="noopener" href="mailto:jobs@mycrypto.com">jobs@mycrypto.com</a> and include the words “DESIGNER” in the subject line:</h5>
-      <ul>
-        <li>Your resume (if an up to date one is available).</li>
-        <li>Any online portfolios.</li>
-        <li>Examples of things you have really, really enjoyed working on.</li>
-        <li>Some not-boring information about yourself and what you love.</li>
-        <li>What you find attractive about this potential job / working for MyCrypto.</li>
-      </ul>
-    </div>
-  </div>
-
-
-
+    </ul>
+    
+    <h5>Things We Don’t Care About</h5>
+    <ul>
+    <li>When you sleep or work, where you are located, what you do in your free time, your gender, age, race, sexual preference, or websites you only open in incognito mode.</li>
+    </ul>
+    
+    <h5>React / Javascript / Typescript Engineers</h5>
+    <p>Specifically, we are currently looking for amazing React / Javascript / Typescript engineers to work on MyCrypto.com. We have a <a target="_blank" rel="noopener" href="https://github.com/MyCryptoHQ/MyCrypto/tree/gau">new update</a> that is pretty huge and includes multi-account support and endless opportunities for unique features. We hope that you have worked in this space before and/or are passionate about the blockchain. Our products rarely have a traditional back-end, but that doesn't mean this is a strictly a "front-end" or "styling" job. It requires deep, client-side logic, an understanding of how to interact with the blockchain on a technical level, and the ability to figure shit out without great documentation because, well, this is crypto and apparently documentation comes second to changing the world. 😉</p>
```

Comment:
These two lines don't feel like they belong in the "Things we don't care about" section.

Reply:
The `h5` means its own section - so it's in the *React / Javascript / Typescript Engineers* section